plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -89,6 +93,7 @@ public void start()
                 .build();
         s3Client.createBucket(this.bucketName);
```

Comment:
Why do we have both s3client and minioclient?
I think that the s3client is now superfluos.

Reply:
👍  for that. One or another. My initial idea was to use S3Client as this would allow to have some base class in future which would handle real S3 tests and Minio based test.
For now it seems we should stick to one selected client.